src/frontend/app/shared/components/cards/service-broker-card/service-broker-card.component.html

```diff
@@ -0,0 +1,28 @@
+<app-meta-card class="service-broker">
+  <app-meta-card-title>
+    <div class="service-broker__title__link">{{ (serviceBroker$ | async)?.entity.name }}</div>
```

Comment:
The title should move down to the main list (`Name: <x>`) and be replaced here with `Service Broker`.